docker/php/docker-entrypoint.sh

```diff
@@ -21,6 +21,7 @@ if [ "$1" = 'php-fpm' ] || [ "$1" = 'php' ] || [ "$1" = 'bin/console' ]; then
 		composer create-project "$SKELETON $SYMFONY_VERSION" tmp --stability="$STABILITY" --prefer-dist --no-progress --no-interaction --no-install
 
 		cd tmp
+		composer require "php:$PHP_VERSION"
```

Comment:
With `php:>=$PHP_VERSION` would be better. Please, can you modify it?

Reply:
Done